docs/user_guide/analytics.rst

```diff
@@ -1,6 +1,57 @@
 Analytics and usage services
 ============================
 
+The theme supports several web analytics services via the ``analytics`` option. It is configured
+by passing a dictionary with options. See the sections bellow for relevant
+options depending on the analytics provider that you want to use.
+
+.. code:: python
+
+   html_theme_options = {
+       # See below for options for each service
+       "analytics": analytics_options,
+   }
+
+Generally speaking we recommend using Plausible over Google Analytics because
+it has a better story around user security and privacy. In addition, it is more
+open-source and transparent. In fact,
+`you can self-host a Plausible server <https://www.elvisduru.com/blog/how-to-self-host-plausible-analytics>`__.
+
+.. admonition:: Get a self-hosted Plausible server at ``scientific-python.org``
+   :class: tip
+
+   If your documentation is for a package that is part of the SciPy / PyData
+   ecosystem, they might be able to host a Plausible server for you at
+   ``<your-package>.scientific-python.org``.
```

Comment:
@tupui Note that we host all projects at `views.scientific-python.org`

Reply:
Is there anything project-specific in a URL? Or just one big dashboard for everybody?

Reply:
Ah yes it should be `https://views.scientific-python.org/<your-package>`